static/styles/app_components.css

```diff
@@ -1,7 +1,7 @@
 /* Reusable, object-oriented CSS base components for the Zulip webapp
    (not included in the portico CSS) */
 
-   .flex {
+.flex {
```

Comment:
Ahh, this fixup is just in the wrong commit -- it's important to read your commits with a tool like `gitk` and make sure you put fixes into the right commit.